src/Microsoft.Azure.WebJobs.Host/Scale/IScaleMetricsRepository.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Interface defining methods for reading/writing scale metrics to a persistent store.
+    /// </summary>
+    public interface IScaleMetricsRepository
```

Comment:
In the ScaleController we won’t be persisting metrics to storage. Is the idea that SC would use in-memory, as a replacement for current scale controller metrics capturing?

Reply:
yes, SC will be using in-memory implementation.